docs/quick_start/starting.rst

```diff
@@ -47,6 +53,44 @@ It is also possible to use certificate based authentication to login to the syst
 
 .. note::
 
-   Extreme caution should be taken when storing certificate files that they not be readable by any other user within the system.
+   Extreme caution should be taken when storing certificate files. They must not be readable by any other users within the system.
 
 Following a successful authentication attempt a time based token is issued that allows the user to run further commands, for a limited time, without the need to authenticate again.
+
+Authentication File
+-------------------
+
+The prompting for username and password when using the *fledge* script can be bypassed if an authentication file is created. This is a file that should be created in a directory called *.fledge* in the user's home directory.
+
+The file created should be called *auth* and contains the credentials required to login. This may either be a username and password or the filename of a certficate to use to authenticate.
+
+.. note::
+
+   The *auth* file will only be read if the permissions on that file are set such that only the owner can read the file.
+
+   .. code-block:: console
+
+       $ chmod 600 ~/.fledge/auth
+
+   In older versions of Fledge the *auth* file was simply called *~/.fledge*. If the older *.fledge* file exists it will still be used.
+
+An example *auth* file, using the default username and password would be as follows
+
+.. code-block:: console
+
+   FLEDGE_USER=admin
+   FLEDGE_PASSWORD=fledge
+
+If using a certificate to authenticate the file would look as follow
+
+.. code-block:: console
+
+   FLEDGE_CERT=~/.auth/user.cert
```

Comment:
Are we allowing non systemctl role users as well? and expect them to copy their cert to this path?

Reply:
Yes, if they are using the fledge script then they are by definition Linux users so are able to do this. They don't have to of course, they can use any of the other methods to authenticate.